crates/solidity/inputs/schema/grammar/04-statements/01-blocks/productions.yml

```diff
@@ -2,24 +2,35 @@
 
 - name: "Block"
   kind: "Parser"
-  unversioned:
-    delimitedBy:
-      open:
-        reference: "OpenBrace"
-      expression:
-        zeroOrMore:
-          choice:
-            - reference: "Statement"
-            - reference: "UncheckedBlock"
-      close:
-        reference: "CloseBrace"
+  versioned:
+    0.4.11:
+      delimitedBy:
+        open:
+          reference: "OpenBrace"
+        expression:
+          zeroOrMore:
+            reference: "Statement"
+        close:
+          reference: "CloseBrace"
+    0.8.0:
+      delimitedBy:
+        open:
+          reference: "OpenBrace"
+        expression:
+          zeroOrMore:
+            choice:
+              - reference: "Statement"
+              - reference: "UncheckedBlock"
+        close:
+          reference: "CloseBrace"
 
 - name: "UncheckedBlock"
```

Comment:
TOL, I wonder why `UncheckedBlock` wasn't added as a choice under `Statement`, perhaps `UncheckedStatement`?

I see it as a natural sibling of `WhileStatement`, `IfStatement`, etc... Thoughts?